Src/FluentAssertions/Events/OccurredEvent.cs

```diff
@@ -21,5 +21,10 @@ public class OccurredEvent
         /// The exact date and time of the occurrence in <see cref="DateTimeKind.Local"/>.
         /// </summary>
         public DateTime TimestampUtc { get; set; }
+
+        /// <summary>
+        /// The order in which this event was raised on the monitored object.
+        /// </summary>
+        public int RaisedOrderIndex { get; set; }
```

Comment:
🤔 Just like my previous comments, I think this should be called `Sequence`. Same with the other properties with the same purpose.

Reply:
Done in https://github.com/fluentassertions/fluentassertions/pull/1773/commits/ee0b86ba673e5491c542e9aca797095ad66db2cc along with other properties.